src/components/OrgListCard/OrgListCard.module.css

```diff
@@ -21,7 +21,7 @@
   border-radius: 4px;
 }
 
```

Comment:
.orgCard .innerContainer .orgImgContainer .emptyImg {
  width: 125px;
  height: 120px;
  background-color: var(--bs-gray-200);
}

You removed this class from the UI but did not remove this from the CSS file